src/connection-stage-actions.ts

```diff
@@ -172,9 +172,6 @@ export class ConnectionStageActions {
   };
 
   onChangeMicrobitName = (name: string) => {
-    if (this.stage.connType !== "bluetooth") {
-      throw new Error("Microbit name can only be set for bluetooth flow");
-    }
```

Comment:
Validation is wrong. It should be checking `flowType`, not `connType`. But also, this scenario should not be possible anyway because radio dialogs cannot trigger this callback, so I have removed this if-statement.